hmt_escrow/job.py

```diff
@@ -124,6 +124,50 @@ def manifest_hash(escrow_contract: Contract,
     })
 
 
+def intermediate_url(escrow_contract: Contract,
+                     gas_payer: str,
+                     gas: int = GAS_LIMIT) -> str:
+    """Retrieves the deployed intermediate results url uploaded on Job initialization.
+
+    Args:
+        escrow_contract (Contract): the escrow contract of the Job.
+        gas_payer (str): an ethereum address paying for the gas costs.
+        gas (int): maximum amount of gas the caller is ready to pay.
+    
+    Returns:
+        str: returns the intermediate results url of Job's escrow contract.
+
```

Comment:
want a doctest here?  (might not be worth it for something this small, just provoking the idea)